x-pack/plugins/security_solution/public/cases/components/all_cases/index.test.tsx

```diff
@@ -97,23 +96,16 @@ describe('AllCases', () => {
   });
   /* eslint-enable no-console */
   beforeEach(() => {
-    jest.resetAllMocks();
+    jest.clearAllMocks();
     navigateToApp = jest.fn();
-    const kibanaMock = createUseKibanaMock()();
-    useKibanaMock.mockReturnValue({
-      ...kibanaMock,
-      services: {
-        application: {
-          navigateToApp,
-        },
-      },
-    });
+    useKibanaMock().services.application.navigateToApp = navigateToApp;
     useUpdateCasesMock.mockReturnValue(defaultUpdateCases);
     useGetCasesMock.mockReturnValue(defaultGetCases);
     useDeleteCasesMock.mockReturnValue(defaultDeleteCases);
     useGetCasesStatusMock.mockReturnValue(defaultCasesStatus);
     moment.tz.setDefault('UTC');
   });
+
```

Comment:
Within the codebase there's like a weird style issue between newlines and not newlines between blocks of it and describe. No changes requested, just odd when I see two different styles and almost want a linter rule to happen at some point to just make it one way.